pkg/rules/rules_test.go

```diff
@@ -82,7 +82,7 @@ func testRulesAgainstExamples(t *testing.T, dir string, server rulespb.RulesServ
 		{
 			Name:                    "thanos-sidecar",
 			File:                    filepath.Join(dir, "alerts.yaml"),
-			Rules:                   []*rulespb.Rule{someAlert, someAlert},
+			Rules:                   []*rulespb.Rule{someAlert, someAlert, someAlert},
```

Comment:
@yeya24  I have added this `someAlert` fixtures. I was thinking this is needed here since we are adding an alert to the libsonet.

Reply:
I am not sure. Maybe @kakkoyun or @metalmatze would like to take a look?

Reply:
Ah yes. Sorry for the late reply. I just checked it again and I think this change is needed. Thanks!

Reply:
ok thx!